app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -117,14 +117,14 @@ class ConversationListViewModel @Inject constructor(
             when (it.conversation.mutedStatus) {
                 MutedConversationStatus.AllAllowed ->
                     when (it) {
-                        is Group -> it.unreadMessagesCount > 0
-                        is OneOne -> it.unreadMessagesCount > 0
+                        is Group -> it.unreadMessagesCount > 0 && it.lastUnreadMessage?.senderUserId != selfUser?.id
```

Comment:
I think that checking only the last message isn't enough, I think we should modify the query to only count messages as unread if they come from someone else